packages/frontend/editor-ui/src/features/dataStore/components/dataGrid/DataStoreTable.vue

```diff
@@ -184,17 +168,38 @@ const onDeleteColumn = async (columnId: string) => {
 	}
 };
 
+const onAddColumn = async ({ column }: { column: DataStoreColumnCreatePayload }) => {
+	try {
+		// todo: api call should be last
```

Comment:
Should this TODO be addressed?

Reply:
I should probably remove the comment instead. I haven't changed this method but was thinking from UX perspective that we want to quickly update the UI first and then do the api call (optimistic) but didn't consider that we wouldn't have the id so I guess for now we've decided to go without the optimistic update and this comment is not needed.